.github/pull_request_template.md

```diff
@@ -20,8 +20,12 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 ### Check list  
 * Is your feature branch up to date with the authoritative repository (NOAA/develop)?
 * Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
+* Make sure you have added labels to the PR, including but not limited to _bug_, _documentation_, _duplicate_, _enhancement_, _invalid_, _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, _new feature_, _Regression test_, _UFS_, ...
```

Comment:
Before adding this, we should check that someone without write permissions can in fact add labels.

Reply:
the changes could be questions in the regression tests part as that's more closely linked to that.

Reply:
moved there